src/server_state.cpp

```diff
@@ -612,6 +612,9 @@ void wsrep::server_state::sst_received(wsrep::client_service& cs,
                 throw wsrep::runtime_error(msg.str());
             }
 
+            if (current_view_.status() == wsrep::view::primary) {
```

Comment:
Please follow the style placing the curly bracket after `if ()` on separate line.